scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -932,7 +937,10 @@ import java.net.{ServerSocket, InetAddress}
    * @return a <code>Runner</code> implementation representing the newly started run to run ScalaTest's tests.
    * @throws IllegalArgumentException when invalid or unsupported argument is passed
    */
-  def runner(args: Array[String], remoteArgs: Array[String], testClassLoader: ClassLoader): SbtRunner = {
+  def runner(_args: Array[String | Null] | Null, _remoteArgs: Array[String | Null] | Null, _testClassLoader: ClassLoader | Null): SbtRunner = {
+    val args = _args.nn.map(_.nn)
+    val remoteArgs = _remoteArgs.nn.map(_.nn)
+    val testClassLoader = _testClassLoader.nn
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.